### PR TITLE
restrict visibility on filegroup target only consumed in 1 place

### PR DESCRIPTION
**Pull request by thundergolfer**
Reviewd #601 after it merged. Thanks for fixing this long outstanding annoyance 🙂  @UebelAndre 

If I understand this code correctly the filegroup is only consumed by `//python/pip_install/private:srcs_module` and so you don't need public visibility on the target.

### CODE REVIEW

**Comment by UebelAndre** [comment]
Thanks for the extra review and sorry about the miss. I'll be sure to pay more attention to visibility going forward 😀

**Comment by thundergolfer** [comment]
@UebelAndre no worries at all, and don't worry about it. It's such a minor thing 🙂

File: python/pip_install/BUILD [hunk 1]
```diff
@@ -26,7 +26,7 @@ filegroup(
         "//python/pip_install/extract_wheels:py_srcs",
         "//python/pip_install/parse_requirements_to_bzl:py_srcs",
     ],
-    visibility = ["//visibility:public"],
+    visibility = ["//:__subpackages__"],
```
Comment (f0rmiga) [inline python/pip_install/BUILD:29]:
```suggestion
    visibility = ["//python/pip_install/private:__pkg__"],
```

**Review by f0rmiga — APPROVED** [review]
Since it's only used by `//python/pip_install/private:srcs_module`, as stated in the PR description, we could be even more diligent.

I'll leave it up to you to take the suggestion or not.

**Pull request merged**